getting-started.adoc

```diff
@@ -1,154 +1,192 @@
+
+Raincatcher consists of a set of reusable modules that can be included in your application using http://browserify.org/[browserify]. 
+
+
+Raincatcher is based on the principle of multiple,
+loosely-coupled modules communicating through the mediator pattern.
+Raincatcher modules make use of the
+https://addyosmani.com/largescalejavascript/[mediator pattern] to enable
+loose coupling between the Raincatcher modules and their consuming
+applications / modules as shown in the following diagram:
+
+image:assets/images/wfm-architecture.png[Raincatcher Architecture]
 
-Raincatcher is a field workforce management (WFM) application that connects a business' back-office with its fleet of mobilized employees.  Raincatcher leverages commoditized mobile hardware, making use of each employee's mobile phone.
 
 [[raincatcher-structure]]
-Raincatcher Structure
-~~~~~~~~~~~~~~~~~~~~~
+== Raincatcher Structure
 
-Raincatcher consists of a set of reusable modules that are designed to be included in your application using [browserify](http://browserify.org/). Raincatcher modules are packaged and distributed via [npm](https://www.npmjs.com/) and export one or more of:
+Raincatcher modules are packaged and distributed via [npm](https://www.npmjs.com/) and export:
 
-* Angular.js directives or services providing client-side functionality
-for both the mobile and portal clients.
+* Angular.js directives or services providing client-side functionality for both the mobile and portal clients.
 * Express.js routes providing a REST API to be consumed by the
 client-side portions of the module.
-* FeedHenry sync configurations for enabling data synchronization of a
-module’s data.
-
-Refer to the READMEs of the respective modules for details on their
-purpose and consumption.
+* FeedHenry sync configurations which enable data synchronization of  module data.
 
-In addition to this, a demo application is provided as a project template that showcases how these modules can be assembled into an application. See [this guide](the-raincatcher-demo-solution-architecture) for the demo application architecture.
+A demonstration is also provided as a project template that shows how the modules can be assembled into an application. See [this guide](the-raincatcher-demo-solution-architecture) for the demo application architecture.
 
 [[running-the-raincatcher-demo-apps-in-rhmap]]
-Running the Raincatcher Demo Apps In RHMAP
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-RHMAP provides a set of Raincatcher project templates to help you get
-the Raincatcher demo apps up and running.
-
-[[create-and-configure-the-raincatcher-demo-project-using-the-project-templates]]
-Create and Configure the Raincatcher Demo Project Using the Project
-Templates
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-1.  Create a blank Forms theme.
-2.  Navigate to **Drag & Drop Apps -> Forms Themes**.
-image:assets/images/select-forms-themes.png[Select Forms Themes]
```

Comment:
@pmdarrow  I started working on your comments and got stuck at this one. It's true that screenshots are helpful to the novice user, but they lose their value over time as the solution develops and the screens become out of date. I'll follow up with an email on how we might get the best of both worlds, screenshots and low-maintenance doc.